src/components/DopplerPlus/PlanCalculator.js

```diff
@@ -36,9 +47,32 @@ const PlanCalculator = () => {
               {/* TODO: change this to intl elemnt */}
               ¿Cuántos contactos tienes? Utiliza el slider para calcular el costo final de tu Plan
             </p>
+            <div style={{ marginBottom: '40px' }}>
+              {discountsList.map((discount, index) => (
+                <button
+                  key={index}
+                  style={{
+                    padding: '10px',
+                    border: '1px solid #000',
+                    backgroundColor: discount.id === currentDiscount.id ? 'red' : 'white',
+                  }}
+                  onClick={() => {
+                    applyDiscount(index);
+                  }}
+                >
+                  {discount.description}
+                </button>
+              ))}
+            </div>
             <span>US$</span>
-            <span style={{ fontSize: '40px' }}>{currentPlan.price}</span>
-            <p>por mes</p>
+            <span style={{ fontSize: '40px' }}>
+              {Math.round(
```

Comment:
How about adding the value saved with strikethrough? Like here
![image](https://user-images.githubusercontent.com/2439363/89679659-f47ac700-d8c7-11ea-9ed2-756309aef32b.png)
With less styling of course